src/partner/ducks/pastEvent.ts

```diff
@@ -0,0 +1,60 @@
+import { IEvent } from "src/partner/models/Event";
+
+const FETCH_PAST_EVENT_START = "partner/views/pastEvent/FETCH_PAST_EVENT_START";
+const FETCH_PAST_EVENT_SUCCESS = "partner/views/pastEvent/FETCH_CURRENT_EVENT_SUCCESS";
+const FETCH_PAST_EVENT_FAIL = "partner/views/currentEvent/FETCH_PAST_EVENT_FAIL";
+const UPDATE_EVENT = "partner/views/pastEvent/UPDATE_EVENT";
+
+export interface IState {
+  events: IEvent[];
+  isLoading: boolean;
+  error: any;
+}
+
+export interface IAction {
```

Comment:
rename interface to `IFluxStandardAction` and move it to `partner/ducks/index.ts`